### PR TITLE
Added code of conduct

### PR DESCRIPTION
**Pull request by BlueReZZ**
This will be the canonical source of the code of conduct with all code repos linking to this from their own CODE_OF_CONDUCT.md

### CODE REVIEW

**Comment by thewilkybarkid** [comment]
Might be better on the website?

**Comment by BlueReZZ** [comment]
> Might be better on the website?

That was my original idea but having looked at others (like [kubernetes](https://github.com/cncf/foundation/blob/master/code-of-conduct.md)) and advice from Github and the Contributor Covenant website, having it as an MD file in the community repo seems to be a convention. I suppose it's closer to the code than the governance/CLA type stuff so makes sense.

**Comment by BlueReZZ** [comment]
No reason why we couldn't have https://libero.pub/code-of-conduct/ redirect here too

**Comment by girishmuraly** [comment]
Seems good to me

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at d.moulton@elifesciences.org or girish.nair@hindawi.com. All
+complaints will be reviewed and investigated and will result in a response that
+is deemed necessary and appropriate to the circumstances. The project team is
+obligated to maintain confidentiality with regard to the reporter of an incident.
```
Comment (davidcmoulton) [inline CODE_OF_CONDUCT.md]: 'obliged'?

**Review by davidcmoulton — APPROVED** [review]

**Pull request merged**